src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/BQSRPipelineSpark.java

```diff
@@ -94,7 +94,7 @@ protected void runTool(final JavaSparkContext ctx) {
         final JavaRDD<GATKRead> filteredReadsForBQSR = initialReads.filter(read -> bqsrReadFilter.apply(read));
```

Comment:
You forgot to remove the check above:
    
         // TODO: workaround for known bug in List version of getParallelVariants
         if ( baseRecalibrationKnownVariants.size() > 1 ) {
             throw new GATKException("Cannot currently handle more than one known sites file, " +
                     "as getParallelVariants(List) is broken");
         }